cylc/flow/unicode_rules.py

```diff
@@ -151,3 +152,18 @@ class XtriggerNameValidator(UnicodeRuleChecker):
     RULES = [
         allowed_characters(r'a-zA-Z0-9', '_')
     ]
+
+
+class MessageTriggerValidator:
```

Comment:
Could you inherit from the `UnicodeRuleChecker` and change the `validate` method to be a new "rule" something like:

```suggestion
class MessageTriggerValidator(UnicodeRuleChecker):

    RULES = [
        requires_prefix([
            f'{level}:'
            for level in logging._nameToLevel
        ])
    ]
```

Because the UnicodeRuleChecker class does some docstring wrangling to make its subclasses auto-documenting e.g. https://cylc.github.io/cylc-doc/8.0a2/html/user-guide/suite-name-reg.html#cylc.flow.unicode_rules.SuiteNameValidator).